src/client/components/Hello.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
Hi, the project already has one `component` folder inside the common folder from the root, I made that because this project is having `SSR` so it will take that `component` from there. So I think `component` inside the client is fine which you did, may be we can discuss common component later, we need to delete the component folder from the `common`, and then need to change the path where it was used (in the server and in the `src/server/controller`)

so can you make that change or after merging this I will do it ? your call !

Reply:
I know, I've seen Your SSR-related code. The server-side is up to You, so You decide. However, as far as I know, it's not uncommon to finish the client-side first, and then proceed to implementing the `common` file for SSR-related purposes. [The SSR thing is v. ambitious btw.]

Reply:
okay ! then I will change the SSR path after this commit. no issue 😃